docs/source/_templates/sdk_submit_tutorial.rst

```diff
@@ -144,6 +164,18 @@ Optionally, you may pass in header properties in order to override any defaults
 
 {% else %}
 
```

Comment:
Question (more than a comment) regarding the text above this code: what if an encoder function is **not** set? Would this cause an error, or is there another way to get the transaction payload encoded? If an error will be generated if an encoder function is not set, maybe we should state that it is a requirement to set an encoder method...